.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: csharp
 mono: none
 dist: trusty
```

Comment:
Builds with `trusty` as dist will currently fail. You could use `xenial` temporary: https://travis-ci.community/t/dotnet-core-2-2/1216

Reply:
It looks like Xenial is a later release https://wiki.ubuntu.com/Releases
should probably bump up to Xenial (`16.0`) or Bionic (`18.0`) anyway.
Ubuntu 18 is what I run locally in WSL anyway.

Reply:
Tried `bionic`. It's up to date and t works, so we're good :)